### ISSUE TITLE
Add trial reel option 

### ISSUE THREAD

**Issue by mrafieefard** [subzeroid/instagrapi#2063]
Recently Instagram added a feature for new account to upload their reels with trial option
Trial option will show your reel to high amount of people 

![image](https://github.com/user-attachments/assets/12c509f2-ee40-47fb-8414-49b533230627)

It would be great if you add this feature to uploading reels

**Comment by Fire162** [comment]
**Yes I Also Want This Feature**

**Comment by huber628** [comment]
Yes this feature is crazy good, I hope to see it in the future in instagrapi too

**Comment by subzeroid** [comment]
Closing this as a duplicate of the canonical Trial Reels request: https://github.com/subzeroid/instagrapi/issues/2145

The feature remains open there and is tracked as capture-needed until the current app Trial Reels publish flow is verified.

**Comment by subzeroid** [comment]
Trial Reel upload support has now landed in https://github.com/subzeroid/instagrapi/pull/2471.

Use `clip_upload(..., trial=True)` to send `trial_params={"graduation_strategy": "manual"}` and hide the feed preview for the Trial Reel upload. Note that Instagram still gates Trial Reels by account eligibility server-side.

**Comment by subzeroid** [comment]
Related upload changes were released in `2.5.17`: https://github.com/subzeroid/instagrapi/releases/tag/2.5.17

PyPI: https://pypi.org/project/instagrapi/2.5.17/
